lesson02/assignment/css/_components.content.scss

```diff
@@ -1,36 +1,48 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Lesson 02 - Assignment 02!</title>
-    </head>
-    <body>
-        <main class="main">
-          <div class="main_h1">
```

Comment:
You'll want to add an element or two to wrap your card to create the outer gray background color/border. Take a look at getting that in there, and re-add me as a reviewer.

Reply:
Hi Bryce,

When you say add an element or two to create that outer gray background, is that different from the DIV class, "section", that I've created and stylized in the CSS file? Sorry, I'm a bit confused what you mean by this.

Thank you.

Reply:
Hi Alex,

While you've got the white bg around the component and the gray bg around the details, the surrounding gray background/border isn't present. I was saying you can wrap your current component in another element (like a div) and style that to make the outer bg/border. Does that make sense?

Thanks,
Bryce